bedrock/settings/__init__.py

```diff
@@ -47,6 +47,8 @@
     "region1.google-analytics.com",
     "telemetry.transcend.io",  # Transcend Consent Management
     "telemetry.us.transcend.io",  # Transcend Consent Management
+    "cdn.transcend.io",  # Transcend Consent Management
```

Comment:
Added to `_csp_connect_src` to resolve this [Sentry report](https://mozilla.sentry.io/issues/7106735964/?project=6260233&query=is%3Aunresolved&referrer=issue-stream)